mosviz/utils/cutout_tools.py

```diff
@@ -34,6 +34,8 @@ def make_cutouts(catalogname, imagename, image_label,
         * ``'dec'`` - DEC (e.g., in degrees).
         * ``'cutout_x_size'`` - Cutout width (e.g., in arcsec).
         * ``'cutout_y_size'`` - Cutout height (e.g., in arcsec).
+        * ``'cutout_pa'`` - Cutout angle (e.g., in degrees). This is only
```

Comment:
What does 'e.g.' mean in these examples? Are they required to be in those units?